src/main/java/com/unityTest/courseManagement/entity/AssignmentEnrollment.java

```diff
@@ -30,10 +34,10 @@
 	private int id;
 
 	// Assignment user is enrolled in
-	// @ApiModelProperty(value = "Enrolled assignment")
-	// @ManyToOne(fetch = FetchType.LAZY)
-	// @JoinColumn(name = "ASSIGNMENT_ID", referencedColumnName = "ID")
-	// private Assignment course;
+	@ApiModelProperty(value = "Enrolled assignment")
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "ASSIGNMENT_ID", referencedColumnName = "ID")
+	private Assignment course;
```

Comment:
I forgot to rename this, oops
```suggestion
	private Assignment assignment;
```